### PR TITLE
[WIP] FSharpQA tests migration to xUnit suite - Part 1/?

### PR DESCRIPTION
**Pull request by vzarytovskii**
This is a (semi-)automatic migration of F#QA tests to xUnit suite.
This PR is migrating only small fraction of simpler tests, which do not require running, IL comparison, or linking other libs - to test the migration script.

After this one is passing, I will start migrating other batches.

### CODE REVIEW

File: tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj [hunk 1]
```diff
@@ -58,7 +204,4 @@
     <ProjectReference Include="$(FSharpTestsRoot)\FSharp.Test.Utilities\FSharp.Test.Utilities.fsproj" />
```
Comment (dsyme) [inline tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj:204]: These tests can directly use FSharp.Compiler.Service.fsproj, and the test project can be included in FSharp.Compiler.Service.sln if not there already

File: tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj [hunk 2]
```diff
@@ -19,9 +19,10 @@
   </ItemGroup>
```
Comment (dsyme) [inline tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj:19]: These tests can run just on `netcoreapp3.1`.  We have enough coverage on `net472` through visual studio tests and other `test/fsharp` tests

Comment (KevinRansom) [reply tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj:19]: @dsyme This is the fsharpqa suite.  For sure they run on desktop, if we can't make them run net472 then we are doing something wrong.

**Comment by vzarytovskii** [comment]
I've significantly reduced tests migrated to the xunit - only test which don't require running or IL comparisons. Migraing them in smaller batches makes troubleshooting so much easier.

**Review by KevinRansom — APPROVED** [review]

**Pull request merged**